src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java

```diff
@@ -164,8 +166,11 @@ protected void configure() {
             withEnv("KC_FEATURES_DISABLED", String.join(",", featuresDisabled));
         }
 
-        withEnv("KC_BOOTSTRAP_ADMIN_USERNAME", adminUsername);
-        withEnv("KC_BOOTSTRAP_ADMIN_PASSWORD", adminPassword);
+        if(bootstrapAdmin) {
```

Comment:
Please add a space before the brace to meet existing code formatting in this repo.

Reply:
Well spotted, done